_styleguides/de.md

```diff
@@ -0,0 +1,557 @@
+# Stilregeln für die Übersetzung von Godot ins Deutsche
+
+Originalautor: Cerno_b (cerno.b@gmail.com)
```

Comment:
I don't think we do this kind of attribution anywhere else, we just depend on the Git history and PRs to show authorship.

Reply:
Sure, I'll remove it. I thought it would be a good idea for questions, but you're right, this can be seen in git.

Reply:
done